shared/reducers/tabbed-router.js

```diff
@@ -34,7 +35,10 @@ export default function (state: TabbedRouterState = initialState, action: any):
   switch (action.type) {
     case Constants.switchTab:
       return state.set('activeTab', action.payload)
-    default:
+    case RouterConstants.navigateUp:
```

Comment:
all actions go to all reducers so we shoudln't do this work unless it actually has to do with the sub reducer